src/components/Dashboard/Main.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import React, { Component, Fragment } from 'react';
+import { bindActionCreators } from 'redux';
+import {
+  Flavors,
+  DashboardHome as Home,
+  Migrations,
+  DashboardNotFound as NotFound,
+  Roles,
+  RoleAdd,
+  RoleEdit,
+  RoleDelete,
+  RoleUsers,
+  RoleAddUser,
+  RoleDeleteUser,
+  Users,
+  UserRoles
+} from 'components/Dashboard/';
+
+import { actions as dashboardActions } from 'reducers/dashboard';
+import { getDashboardComponent } from 'selectors/dashboard';
+
+class DashboardMain extends Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    dashboardComponent: PropTypes.object.isRequired
+  };
+  display() {
```

Comment:
You can make this a getter, but you might as well just fold it into the render method since render isn't doing anything else.

Reply:
Fixed